src/balance.js

```diff
@@ -1,17 +1,33 @@
 const { BN } = require('./setup');
 
-async function balanceCurrent (account) {
-  return new BN(await web3.eth.getBalance(account));
+class Tracker {
+  constructor (acc) {
+    this.account = acc;
+    this.balances = [];
```

Comment:
Using an array seems a bit overkill for this, since we only need to store the previous balance (to calculate `delta`). That way, we'd get rid of the `push`, `shift` and `length` calls, resulting in much simpler code. Something like:

```
get() {
  this.prev = current();
  return this.prev;
}

delta() {
  const current = current();
  const delta = current - this.prev;
  this.prev = current;
  return delta;
}
```